freqtrade/freqtradebot.py

```diff
@@ -488,7 +516,7 @@ def get_real_amount(self, trade: Trade, order: Dict, order_amount: float = None)
         if order_amount is None:
             order_amount = order['amount']
         # Only run for closed orders
-        if trade.fee_open == 0 or order['status'] == 'open':
+        if trade.fee_open == 0 or order.get('status', None) == 'open':
```

Comment:
is it possible for an order to be with undefined status?

Reply:
Impossible now, this must be leftover from some earlier changes